components/modal/Modal.tsx

```diff
@@ -48,9 +48,11 @@ const Modal = () => {
       className={`flex items-center justify-center fixed  rounded  ${positionCommon} ${transitionOptions} ${isOpen ? 'opacity-100' : 'opacity-0 pointer-events-none'}`}
     >
       <div
-        className={`modal-backdrop fixed ${positionCommon} bg-violet-500 bg-opacity-50`}
+        className={`modal-backdrop fixed ${positionCommon} bg-violet-500 bg-opacity-50  ${isOpen ? 'block' : 'hidden'}`}
       />
-      <div className="modal-wrapper w-[calc(100%-48px)] h-[calc(100%-48px)] px-[50px] py-[50px] z-40">
+      <div
+        className={`modal-wrapper w-[calc(100%-48px)] h-[calc(100%-48px)] px-[50px] py-[50px] z-40 ${isOpen ? 'block' : 'hidden'}`}
```

Comment:
is `modal-wrapper `an identifier? if yes, do we need it?

Reply:
I find the lack of component names disturbing for deployment, testing, bugFixing. The "Smartive way" avoids to have them but, I dont want to work for smartive. I want to work only smart. 
We could remove them as per final task. Even so, improving css later on, can be really pain in the while inspecting the markup on the developer tools.

Reply:
I only wanted to know what purpose this string has. I think it is a good idea to keep it for debugging purposes. Don't think we should go through the code and remove those classes.